.github/workflows/build.yml

```diff
@@ -72,11 +72,12 @@ jobs:
           java-version: 17
 
       - name: Publish Artifacts
-        run: ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-gradle-plugin:publishAllPublicationsToMavenCentralRepository --no-parallel
+        run: ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-gradle-plugin:publishAllPublicationsToMavenCentralRepository
```

Comment:
The parallel problem has been fixed for a few years now

Reply:
We still saw problems with this as recently as a year ago, but if we're asking _that_ has been fixed since then, sgtm!